.gitlab-ci.yml

```diff
@@ -17,6 +17,17 @@ build-images:
     - docker build -t ${DOCKER_IMAGE_TAG} --build-arg GIT_SHA=${CI_COMMIT_SHA} --pull .
     - docker push ${DOCKER_IMAGE_TAG}
 
+build-jupyter:
```

Comment:
This  section seems disconnected from the rest of the file. It does things in a different way, e.g. using compose, not tagging the built images, not using the set variables.

Reply:
Good points, I've updated it in c09c0d1 to be more in alignment with the rest of the file.